### PR TITLE
Add document for creating new repository (fixes #26)

### LINKED ISSUE #26: Create a GitHub repository for l10n project

### PR DESCRIPTION
**Pull request by flodolo**
Fixes #26

### CODE REVIEW

File: misc/creating_new_repository.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Creating a new repository in the [mozilla-l10n](https://github.com/mozilla-l10n) organization provides the advantage of letting l10n-drivers manage this repository directly, reducing delays when dealing with permissions. It also allows localizers to work directly on GitHub with pull requests.
+
+Note that you need to be a manager of the organization in order to create a new repository.
+
+## Create the new repository
+A new repository can be created in [this page](https://github.com/organizations/mozilla-l10n/repositories/new), or using the **New** button in the organization’s landing page.
+
+**Name**: always include the `-l10n` suffix for localization repositories, to differentiate them from other repositories used for tools. Notable exceptions to this rule are `appstores` and `www.mozilla.org`.
+
+For example, if the project is *Focus for Android*, a good repository name is `focus-android-l10n`.
```
Comment (mathjazz) [inline misc/creating_new_repository.md:11]: Are we making Focus for Android?

**Review by mathjazz — APPROVED** [review]
Looks great!

File: misc/creating_new_repository.md (hunk 1 above)
Comment (flodolo) [reply misc/creating_new_repository.md:11]: Yep :-) 
https://github.com/mozilla-l10n/focus-android-l10n/

a3807e09afab6d37ff5352a270a467b4

**Comment by flodolo** [comment]
Thanks for the review 👍

**Pull request merged**